Examples/Swift/CustomStyleExample.swift

```diff
@@ -6,8 +6,8 @@ class CustomStyleExample_Swift: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        // Fill in the next line with your style URL from Mapbox Studio.
-        // <#mapbox://styles/userName/styleHash#>
+        // Replace the string in the URL below with your custom style URL from Mapbox Studio
```

Comment:
Nit: can you put a period at the end of this sentence? I like to think it makes it easier to read when wrapped.